generators/app/index.js

```diff
@@ -55,6 +55,10 @@ function input() {
             {
                name: 'Widget dashboard',
                value: 'widget'
+            },
+            {
+                name: 'Service hook',
```

Comment:
I would call this a "Service hooks consumer" so it's clear what is being contributed.